src/account.ts

```diff
@@ -106,6 +106,21 @@ export abstract class Account {
         return this.starknetContract.address;
     }
 
+    async getAccountBalance(): Promise<StringMap> {
+        const abiPath = process.cwd() + "/contract-artifacts/ERC20/ERC20_abi.json";
+
+        const hre = await import("hardhat");
+        const ethContract = new StarknetContract({
+            abiPath: abiPath,
+            hre: hre
+        });
+        ethContract.address = "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7";
+
+        return (
+            await ethContract.call("balanceOf", {account: this.starknetContract.address})
```

Comment:
The value should be extracted and converted to Bigint (this answers the "open question" from the PR description). Even though starknet-hardhat-plugin relies on an older version of starknet.js, even that one should have a utility function for this conversion. We prefer Bigint over Number as it preserves precision.

Reply:
Alright, we return now BigInt. Is there a more direct way of conversion than converting the result first to BigNumber, then to string and finally BigInt?

Reply:
Nice approach, can't think of anything better.